content/docs/user-guide/building/venv.md

```diff
@@ -0,0 +1,50 @@
+# Virtual Environments
+
+Create a python virtual environment using requirements gathered from a model.
+
+## Description
+
+`mlem` can also create virtual environments based on requirements gathered from
+a model. This naturally extends the functionality of the
+[`requirements builder`](/doc/object-reference/build/requirements).
+
+### Generating a new virtual environment
+
+```cli
+$ mlem build model venv -c target="newenv"
+
```

Comment:
Strange, why we have an empty line here? Was it produced by the command?
```suggestion
```

Reply:
no, this was for readability, removed now so as to be consistent.

Reply:
Note: I think it's OK to change stdout for readability. I normally add spaces, truncate lines, replace with `...`, etc. 🙂